service/security/authentication/api/src/main/java/org/eclipse/kapua/service/authentication/credential/CredentialPredicates.java

```diff
@@ -24,5 +24,6 @@
     String CREDENTIAL_TYPE = "credentialType";
     String CREDENTIAL_KEY = "credentialKey";
     String EXPIRATION_DATE = "expirationDate";
+    public static final String MODIFIED_ON = "modifiedOn";
```

Comment:
Please remove this. 

`modifiedOn` predicate is inherited from `KapuaUpdatableEntityPredicates`. 

This change has been introduced yesterday in `develop` so thats maybe why you haven't available before.